exporters/prometheus/src/test/java/io/opentelemetry/exporter/prometheus/PrometheusHttpServerTest.java

```diff
@@ -42,9 +42,9 @@
 import io.opentelemetry.sdk.resources.Resource;
 import io.prometheus.metrics.exporter.httpserver.HTTPServer;
 import io.prometheus.metrics.exporter.httpserver.MetricsHandler;
-import io.prometheus.metrics.expositionformats.generated.com_google_protobuf_4_28_3.Metrics;
+import io.prometheus.metrics.expositionformats.generated.com_google_protobuf_4_29_1.Metrics;
 import io.prometheus.metrics.model.registry.PrometheusRegistry;
-import io.prometheus.metrics.shaded.com_google_protobuf_4_28_3.TextFormat;
+import io.prometheus.metrics.shaded.com_google_protobuf_4_29_1.TextFormat;
```

Comment:
Shading protobuf output with class packages which incorporate the protobuf version means that updating prometheus dependencies will (almost) always require a code change. 

@zeitlinger what are your thoughts on dropping the `_4_29_1` suffix from the package name? I.e.
`io.prometheus.metrics.shaded.com_google_protobuf_4_29_1` -> `io.prometheus.metrics.shaded.com_google_protobuf`.

Reply:
@fstab this is a use case where dropping the suffix would help